mlir-sycl/test/Dialect/IR/SYCL/types.mlir

```diff
@@ -2,7 +2,7 @@
 // RUN: sycl-mlir-opt -allow-unregistered-dialect %s -split-input-file | FileCheck %s
 
 // Verify the generic printed output can be parsed.
-// RUN: sycl-mlir-opt -allow-unregistered-dialect %s -split-input-file | sycl-mlir-opt -allow-unregistered-dialect | FileCheck %s
+// RUN: sycl-mlir-opt -allow-unregistered-dialect %s -split-input-file -mlir-print-local-scope | sycl-mlir-opt -allow-unregistered-dialect | FileCheck %s
```

Comment:
Had to change due to an MLIR bug:
- Aliases are alphabetically sorted, so `vec` alias was placed **below** `swizzled_vec` alias. As `vec` was being used in `swizzled_vec` definition, an "Undefined alias" error was being thrown.

Reply:
That means there should be a way to make an alias depend on a previous alias. Is there any way to indicate that a type alias depends on another type alias?

Reply:
The issue is that aliases are printed in alphabetical order and the `vec` one is defined after the `swizzled_vec` one. I don't think there is a way to change this behavior.